src/olympia/abuse/migrations/0054_add_decision_cinderjob_fk.py

```diff
@@ -0,0 +1,51 @@
+# Generated by Django 4.2.20 on 2025-04-10 21:45
+
+from django.db import migrations, models
+import django.db.models.deletion
+
+
+def set_cinder_job_on_decisions(apps, schema_editor):
+    ContentDecision = apps.get_model('abuse', 'ContentDecision')
+    decisions_with_jobs = []
+
+    # Because there can be a chain of overrides we may need to recurse down the overrides, setting cinder_job on all of them
+    def set_cinder_job(decision, job):
+        decision.cinder_job = job
+        decisions_with_jobs.append(decision)
+        if decision.overridden_by:
+            set_cinder_job(decision.overridden_by, job)
+
+    for first_decision in ContentDecision.objects.filter(_cinder_job__isnull=False):
+        set_cinder_job(first_decision, first_decision._cinder_job)
+
+    ContentDecision.objects.bulk_update(decisions_with_jobs, fields=('cinder_job',))
+
+
+class Migration(migrations.Migration):
+
+    dependencies = [
+        ('abuse', '0053_auto_20250403_1302'),
+    ]
+
+    operations = [
+        migrations.RemoveField(
+            model_name='cinderpolicy',
+            name='default_cinder_action',
+        ),
+        migrations.AddField(
+            model_name='contentdecision',
+            name='cinder_job',
+            field=models.ForeignKey(null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='decisions', to='abuse.cinderjob'),
+        ),
+        migrations.AlterField(
+            model_name='cinderjob',
+            name='decision',
+            field=models.OneToOneField(null=True, on_delete=django.db.models.deletion.SET_NULL, related_name='_cinder_job', to='abuse.contentdecision'),
```

Comment:
this is so we don't clash the related name on `ContentDecision` of the old `CinderJob.decision` with the newly added `cinder_job` field, because we need it in `set_cinder_job_on_decisions`